iceoryx_posh/source/roudi/memory/default_roudi_memory.cpp

```diff
@@ -33,9 +33,9 @@ mepoo::MePooConfig DefaultRouDiMemory::introspectionMemPoolConfig() const
 {
     mepoo::MePooConfig mempoolConfig;
     mempoolConfig.m_mempoolConfig.push_back(
-        {static_cast<uint32_t>(
-             cxx::align(static_cast<uint64_t>(sizeof(roudi::MemPoolIntrospectionTopic)), SHARED_MEMORY_ALIGNMENT)),
-         250});
+        {static_cast<uint32_t>(cxx::align(static_cast<uint64_t>(sizeof(roudi::MemPoolIntrospectionInfoContainer)),
+                                          SHARED_MEMORY_ALIGNMENT)),
+         10});
```

Comment:
Magic number (not just here and was one before). I admit I forgot what it is used for, supposedly some maximum number of those topics in shared memory.  We could at least define a local constexpr to give it an expressive name, such as ```constexpr int32_t magicNumberWhichNeedsABetterName{10};``` and use that. (To make it more readable.)